auth/src/main/java/org/aerogear/mobile/security/AbstractSecurityCheckExecutor.java

```diff
@@ -31,50 +33,64 @@
     private final MetricsService metricsService;
 
     /**
-     * Constructor.
+     * Creates AbstractSecurityCheckExecutor object.
      *
-     * @param context the context
-     * @param checks checks to be executed
+     * @param context the context.
```

Comment:
Specify here if params can be null or not.

Reply:
Is this necessary since the the `NonNull` annotation is added below?

Reply:
Yes, the nonnull annotation is just a lint 'hint', but it does not produce any Javadoc